server/channels/app/support_packet_test.go

```diff
@@ -61,6 +61,9 @@ func TestGenerateSupportPacketYaml(t *testing.T) {
 
 		assert.Equal(t, 3, packet.ActiveUsers) // from InitBasic.
 		assert.Equal(t, licenseUsers, packet.LicenseSupportedUsers)
+		assert.Equal(t, license.Id, packet.LicenseID)
```

Comment:
I think the test license comes with those being empty... so the test is valid, we check that "" == "" haha, just wondering if we can assign values and test against those values.

Reply:
TIL, just fixed it!